apisix/plugins/http-dubbo.lua

```diff
@@ -0,0 +1,242 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+local require = require
+local core = require("apisix.core")
+local pairs = pairs
+local str_format = string.format
+local rshift = bit.rshift
+local band = bit.band
+local char = string.char
+local tostring = tostring
+local ngx = ngx
+local type = type
+local plugin_name = "http-dubbo"
+
+local schema = {
+    type = "object",
+    properties = {
+        service_name = {
+            type = "string",
+            minLength = 1,
+        },
+        service_version = {
+            type = "string",
+            pattern = [[^\d+\.\d+\.\d+]],
+        },
+        method = {
+            type = "string",
+            minLength = 1,
+        },
+        params_type_desc = {
+            type = "string",
+            minLength = 1,
+        },
+        serialization_header_key = {
+            type = "string"
+        },
+        serialized = {
+            type = "boolean",
+            default = false
+        }
+    },
+    required = { "service_name", "method", "params_type_desc" },
+}
+
+local _M = {
+    version = 0.1,
+    priority = 0,
```

Comment:
Is it appropriate to set priority to 0? I think it needs to be discussed.

Reply:
> Is it appropriate to set priority to 0? I think it needs to be discussed.

I think the priority of this plugin is relatively low or even the lowest because it needs to make requests and return results in this stage. Other plugins at the same stage should be executed first before it.

Reply:
fixed